client/common/helpers/PreferenceCreators.jsx

```diff
@@ -0,0 +1,26 @@
+export const optionsOnOff = (name, onLabel = 'On', offLabel = 'Off') => [
```

Comment:
Since `Preferences` is not in `client/common`, I think it makes sense to put this file next to the `Preferences` component. You could even put them in a subfolder together called `Preferences`, and the component could be `index.jsx` and this could be a helper file in that folder.

Reply:
Great idea, will do that